pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -45,13 +45,15 @@ python3.pkgs.buildPythonApplication {
     glib-networking
   ];
 
+  propagatedBuildInputs = lib.optional stdenv.isLinux qtwayland;
```

Comment:
Why is this propagated?

Reply:
because, irrc, it will fail if not. It has been quite some time but if I remember what I did correctly, I originally moved it from build inputs because the wayland backend wouldn't work. It's been there since before this PR, it is only standalone now because I moved all the python dependencies from here to `dependencies`

Unless something has changed or I'm misremembering, you should be able to quickly prove to yourself this is necessary by simply moving it to build inputs and watching it fail when you try to start it on a wayland compositor.

Reply:
> you should be able to quickly prove to yourself this is necessary by simply moving it to build inputs and watching it fail when you try to start it on a wayland compositor.

I did and it works just fine.